app/src/main/kotlin/cz/covid19cz/erouska/ui/login/LoginVM.kt

```diff
@@ -96,12 +100,22 @@ class LoginVM(
             }
 
         }
+    private var showVerifyLaterTimer: CountDownTimer =
+        object : CountDownTimer(AppConfig.showVerifyLaterTimeoutSeconds * 1000, 1000) {
```

Comment:
Tohle je asi NIT, ale imho by mohl byt timeout a interval stejny..ono vlastne by tady slo asi pouzit stary dobry Handler().postDelayed… Ale nemusis to predelavat.